include/h2o/memory.h

```diff
@@ -188,6 +188,10 @@ void *h2o_mem_alloc_recycle(h2o_mem_recycle_t *allocator, size_t sz);
  */
 void h2o_mem_free_recycle(h2o_mem_recycle_t *allocator, void *p);
 
+/**
+ * release all the memory chunks cached in input allocator to system
+ */
+void h2o_mem_allocator_recycle__dispose(h2o_mem_recycle_t *allocator);
```

Comment:
This can be a public function. Please rename it to `h2o_mem_clear_recycle` following our convention (i.e. `h2o_<mobule_name>_verb_...`); "clear" is the term we use for ressetting an object (e.g. `h2o_mem_clear_pool`).

Reply:
yes, fixed